Components/Services/OrderService.php

```diff
@@ -95,6 +99,28 @@ public function getOrderDetailById($orderDetailId)
         return $detail;
     }
 
+    /**
+     * @param string $orderNumber
+     *
+     * @return Transaction
+     *
+     * @throws TransactionNotFoundException
+     */
+    public function getOrderTransactionByNumber($orderNumber)
+    {
+        /** @var Repository $transactionRepository */
+        $transactionRepository = $this->modelManager->getRepository(Transaction::class);
```

Comment:
somehow i feel i've seen a couple of these methods?
could you double check, if its not existing its totally fine

Reply:
![image](https://user-images.githubusercontent.com/13799656/105683948-8f0da780-5ef4-11eb-8ef7-e85a569f1da3.png)

These are all occurences I found. I believe every single one of these occurences differ a bit.

Reply:
sorry i meant the whole new created function to get an order by transaction number.
just checked it, i mixed it with getOrderByNumber, this is getTransactionByNumer, all good